src/Slices/ServiceInstanceDetails/Test/Page.test.tsx

```diff
@@ -202,9 +203,108 @@ describe("ServiceInstanceDetailsPage", () => {
       "Version: 2",
     );
 
+    // There shouldn't be a documentation tab for this Instance and ServiceModel
+    expect(screen.queryByText("Documentation")).not.toBeInTheDocument();
+
     server.close();
   });
 
   // TODO: @LukasStordeur Implement test when config tab has usecases.
   //it("Should render a success view and with config section if present", async () => { });
+
+  it("Should render a success view with documentation", async () => {
+    const server = serverWithDocumentation;
+    server.listen();
+    const component = setup();
+    render(component);
+
+    expect(
+      await screen.findByRole("region", { name: "Instance-Details-Success" }),
+    ).toBeInTheDocument();
+
+    // active attribute set
+    // in this version, we should have documentation available
+    expect(screen.getByText("Latest Version")).toBeInTheDocument();
```

Comment:
I know that is a minor difference, but wouldn't `toBeVisible()` better? Theoretically element could be in the document, but not visible. As we are using patternfly I assume that is very unlikely so it's just a suggestion

Reply:
I can change it, but since those are not elements are either present or not in the DOM, I opted for this assertion.

Reply:
`toBeVisible` has an additional check for `display:none` compared to `toBeInTheDocument`